products/phpbb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: phpBB
+category: server-app
+permalink: /phpbb
+releasePolicyLink: https://www.phpbb.com/community/viewtopic.php?f=14&t=2302466
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: false
+releases:
+-   releaseCycle: "3.3"
+    eol: false
+    latest: "3.3.10"
+    link: https://www.phpbb.com/community/viewtopic.php?t=2635986
+-   releaseCycle: "3.2"
+    eol: 2017-05-01
+    latest: "3.2.11"
+-   releaseCycle: "3.1"
+    eol: 2016-05-01
+    latest: "3.1.12"
+-   releaseCycle: "3.0"
+    eol: 2015-05-01
+    latest: "3.0.14"
+
+---
+
+> [phpBB](https://www.phpbb.com/) is an open-source forum software written in PHP.
```

Comment:
Even if a recent release policy could not be found it would be great to have an overview of what the phpBB release and support policy actually looks like. This overview should document answers to questions such as:

- Does phpBB follows semantic versioning ?
- Is there fixed release cycles or not ?
- What are the supported released ? Only the last X releases ?
- Is there multiple level of support (active, security...) ?

Reply:
It seems [it does follow semantic versioning](https://area51.phpbb.com/docs/dev/master/development/processes.html#target-versions).

I found [this archived page](https://web.archive.org/web/20190627185122/https://www.phpbb.com/about/release/) with more recent information than the post I originally used.
It actually differentiates between "end of maintenance" and "end of life" so I added an active maintenance column.

It seems there was a schedule at some point ("we plan to release a new minor (x.y) or major (x) update once every year around November/December", "EOL occurs to a branch 24 months after its first stable (3.x.0) release"), but now the latest minor release has been 3.3 since 2020.

Reply:
I just noticed the announced EOL date for 3.2 would have been before when 3.3 was actually release, so 3.2 was probably not really EOLed on that date.
However, I could not find a 3.2 EOL announcement :slightly_frowning_face:

Reply:
Congrats finding [this link](https://web.archive.org/web/20190627185122/https://www.phpbb.com/about/release/) ! It seems outdated though, so it may be a good idea to ask on PhpBB support forum (https://www.phpbb.com/community/viewforum.php?f=551).

And if they do not have a release policy anymore we could something similar to:

> phpBB [follows semantic versioning](https://area51.phpbb.com/docs/dev/master/development/processes.html#target-versions). The support policy is not clearly defined, but it seems only the latest minor release sees active development and gets bug and security fixes.